lib/TestRunnerBase.py

```diff
@@ -135,7 +135,7 @@ def _prep_nose_options(self):
         return []
 
     def __get_coverage_packages_opt(self, workdir):
-        with open(self.args.coverage, 'r') as f:
+        with open(os.path.join(workdir, 'tests', 'coverage.json'), 'r') as f:
```

Comment:
Can't we pass the full path to coverage.json to args.coverage? I'm a bit worried about this change.

Reply:
you are right. the change is incorrect actually. I am restoring the change, and need to fix __collect_coverage() as well.  Retesting. Thanks.